recipe/meta.yaml

```
@@ -16,7 +16,7 @@ requirements:
   run:
     - cuda-command-line-tools {{ version }}
     - cuda-visual-tools {{ version }}
-    - gds-tools 1.7.2.10                     # [linux64]
+    - gds-tools 1.8.1.2                     # [linux64]
```

Suggesting keeping the same indentation

```suggestion
    - gds-tools 1.8.1.2                      # [linux64]
```